README.md

```diff
@@ -9,3 +9,11 @@
 3. `yarn install`
 4. `bundle`
 5. `rails db:setup`
+
+If pgcrypto is not installed for a database (and not the postgres user is used):
+
+```
+$ sudo -u postgres psql
+postgres=# \c db_sql_<development|test>;
+postgres=# CREATE EXTENSION IF NOT EXISTS "pgcrypto";
+```
```

Comment:
Maybe, it would also make sense to briefly describe how to create a new db user:

```
sudo -u postgres psql
postgres=# create user foo with encrypted password 'bar';
postgres=# alter role foo with superuser;
```

In addition (I know, this was my fault ^^') I suggest to change line 1 to `cp .env.example .env`